### ISSUE TITLE
No Right Column (Server Info)

### ISSUE THREAD

**Issue by krustymk** [DevelopGravity/Fluid-MC-Stats#22]
Didnt notice this at first due to mainly working in a sub folder, but thinking the layout looked rubbish (single column on the left big nothing on the right)

Maybe connected to the menu button issue?

**Comment by ZacharyDuBois** [comment]
This was the actual intention at the time. On small screens, instead of a two column layout, it will turn into a one column layout to make it look nicer on a small screen. I am however not able to reproduce the issue of the server status information not being displayed at all.

**Comment by ZacharyDuBois** [comment]
(Forgot some labels)

**Comment by krustymk** [comment]
This was on a PC not a tablet. The server info is just not there when installed into a sub folder. The big gap where it should be is though (on a PC monitor, not a tablet) I'll do a screenshot when I get chance

**Comment by krustymk** [comment]
![fluid](https://cloud.githubusercontent.com/assets/5390088/5665010/1026b6c0-974d-11e4-955c-81ce5187a3f1.jpg)

For a while this is how i thought it was supposed to look.. 
http://theamigos.co.uk/stats/ if you want to see it first hand

**Comment by krustymk** [comment]
Also, if you zoom the page (ctrl+mousewheel) till the menu collapses you'll see the 3 lines dont work
Forgive my not knowing the technical names

**Comment by ZacharyDuBois** [comment]
Sorry for the delay, I was sick the past few days. Could you add the following lines to the top of `/inc/status-cache.php` after the includes?

```php
var_dump(APPPATH);
var_dump(is_writable(APPPATH . 'tmp/'));
var_dump(extension_loaded("json"));
die();
```

The top of the file should look like this:

```php
<?php
/**
 * status-cache.php
 * Created for Fluid-MC-Stats.
 */

include_once APPPATH . 'config.php';
include_once APPPATH . 'inc/status.php';
var_dump(APPPATH);
var_dump(is_writable(APPPATH . 'tmp/'));
var_dump(extension_loaded("json"));
die();
```

and change `error_reporting(0);` to `error_reporting(1);` in `inc/header.php`.

This will output a lot of debug information. After doing so, could you screenshot it and post it here?

**Comment by ZacharyDuBois** [comment]
I am not having this issue on my current sub directory install for testing.

**Comment by krustymk** [comment]
![fluid](https://cloud.githubusercontent.com/assets/5390088/5693959/898b8d54-992f-11e4-902b-bac789d5f229.jpg)
Still exactly the same, would you like ftp access to the folder?

**Comment by ZacharyDuBois** [comment]
So the code I told you to insert was to check for a few things that Fluid MC Stats requires that has caused similar issues. The `die()` statement says to stop processing the page there so nothing after should be working when adding that code. But all the values returned are correct. If you would like me to solve this a bit faster, FTP credentials might help. If you do give me access, email them to the email on my GitHub profile and make sure to encrypt them using my [Keybase](https://keybase.io/zachary_dubois) key.

**Comment by ZacharyDuBois** [comment]
@krustymk, could you get back to me on this?

**Comment by krustymk** [comment]
Not been around, ill get on it now

**Comment by krustymk** [comment]
a pm on your bukkit account

**Comment by ZacharyDuBois** [comment]
@krustymk, thanks. I will take a look in a few minutes.

**Comment by ZacharyDuBois** [comment]
It's back up, working on it now.

**Comment by ZacharyDuBois** [comment]
@krustymk, I was able to get it working with the current source. I am not sure why it was not working for you. Debug looks clean too:

```json
{
    "serverInfo": {
        "phpVersion": "5.4.35",
        "webServer": "Apache",
        "os": "Linux",
        "serverName": "theamigos.co.uk",
        "requestURI": "\/stats\/",
        "jsonEnabled": true,
        "mysqliEnabled": true
    },
    "configData": {
        "serverIP": "play.theamigos.co.uk",
        "serverPort": "25565",
        "cdnURI": "",
        "baseURI": "stats\/",
        "avatarURI": "http:\/\/cravatar.eu\/avatar\/",
        "customLinks": [],
        "topStatCalType": "exp",
        "numberOfTopPlayer": "3",
        "hideLimitedWarning": false
    },
    "envInfo": {
        "APPPATH": "\/homepages\/26\/d126148496\/htdocs\/theamigos\/stats\/",
        "LINKBASE": "\/stats\/",
        "fmcsVersion": "A2 v0.2.1"
    },
    "fileInfo": {
        "tmpDirExits": true,
        "tmpDirWrite": true,
        "configExits": true,
        "configWrite": true,
        "installExists": false,
        "queryCacheFileExists": true,
        "queryCacheFileWrite": true
    },
    "otherInfo": {
        "queryData": {
            "mcQuery": {
                "getInfo": false,
                "getPlayers": false
            },
            "mcPing": {
                "query": {
                    "description": "The Amigos Server ... keeping it real",
                    "players": {
                        "max": 50,
                        "online": 3,
                        "sample": [
                            {
                                "id": "b50107f0-ad59-44d7-b77a-4ae85937745b",
                                "name": "Iknoweh"
                            },
                            {
                                "id": "5ac82f18-924e-4957-b10f-155e8ef4dbe1",
                                "name": "Rat_Reaper"
                            },
                            {
                                "id": "02bf0024-21e9-4b45-b5a8-7af349b9ec1e",
                                "name": "Nemah"
                            }
                        ]
                    },
                    "version": {
                        "name": "Spigot 1.8",
                        "protocol": 47
                    }
                }
            },
            "pingTimes": {
                "queryPing": 19,
                "pingPing": 275,
                "avgPing": 147
            },
            "lastUpdate": "2015-01-15 03:50:12"
        }
    }
}
```

You may see the warning in the sidebar. This is a known issue if you don't have UDP opened on your query port. Take a look at issue #20 for some info on those warnings. You can turn off the warning in the config file. Looking at your debug information, it seems your server sends online players, I will have to take a look into adding that :) Also closes #21.
